src/js/service/ImportService.js

```diff
@@ -130,4 +130,75 @@
       return pskl.utils.FrameUtils.createFromImage(resizedImage);
     });
   };
+
+  /**
+   * @param {pskl.model.Piskel} originalPiskel
+   * @param {pskl.model.Piskel} additionalPiskel
+   * @returns {pskl.model.Piskel}
+   */
+  // Adds frames from an additionalPiskel to an originalPiskel, adjusting the size of
+  // both piskels to the largest demensions of either piskel, and centering the content.
+  // This will only work when there is only one layer.
+  // Multi layer animations are disabled from the UI in code-dot-org/piskel
+  ns.ImportService.prototype.addTwoPiskels = function(originalPiskel, additionalPiskel) {
+    var maxWidth = Math.max(originalPiskel.width, additionalPiskel.width);
+    var maxHeight = Math.max(originalPiskel.height, additionalPiskel.height);
+
+    if (additionalPiskel.width <= maxWidth && additionalPiskel.height <= maxHeight) {
+      var additionalFrames = additionalPiskel.layers[0].size();
+      for (var j = 0; j < additionalFrames; j++) {
+        var resizedAdditionalFrame = this.resizeFrameToWH(additionalPiskel.layers[0].getFrameAt(j), maxWidth, maxHeight);
+        additionalPiskel.layers[0].replaceFrameAt(resizedAdditionalFrame, j);
+      }
+      additionalPiskel.width = maxWidth;
+      additionalPiskel.height = maxHeight;
+    }
+
+    if (originalPiskel.width <= maxWidth && originalPiskel.height <= maxHeight) {
+      var originalFrames = originalPiskel.layers[0].size();
+      for (var j = 0; j < originalFrames; j++) {
+        var resizedOriginalFrame = this.resizeFrameToWH(originalPiskel.layers[0].getFrameAt(j), maxWidth, maxHeight);
+        originalPiskel.layers[0].replaceFrameAt(resizedOriginalFrame, j);
+      }
+
+      originalPiskel.width = maxWidth;
+      originalPiskel.height = maxHeight;
+    }
+
+    for (var i = 0; i < additionalPiskel.layers[0].size(); i++) {
+      originalPiskel.layers[0].addFrame(additionalPiskel.layers[0].getFrameAt(i));
+    }
+    return originalPiskel;
+  };
+
+  // Modified from ResizeController
```

Comment:
Since all these functions would have to modified to be used outside of the resize view, I decided to have modified versions here, instead of refactoring to try to keep changes to the fork contained.